Mu2eG4/inc/Mu2eG4WorkerRunManager.hh

```diff
@@ -51,6 +51,7 @@ namespace mu2e {
     void initializeUserActions(const G4ThreeVector& origin_in_world);
     void initializeRun(art::Event* art_event);
     void processEvent(art::Event*);
+    G4Event* generateEvt(G4int i_event);
```

Comment:
Is this a standard G4 function or a Mu2e addition?  The name seems weird to me - it sounds like it is a function that should run an event generator.  So if we control the name, let's talk about a better name.

Reply:
The G4 Worker Run Managers have a member function called generateEvent on which this is based.  The name is related to the fact that the random number seeds are set and the primary particles are generated, in this way the G4 Event is 'made' or generated.  I can change this name if you wish.  I was trying to keep our code as similar to the G4 code as possible so that we understand the relationship between the two.

Reply:
If it's their name then we should use it.  I just needed to know.